Eatery Blue/UI/ListScreen/ListViewController.swift

```diff
@@ -24,6 +24,7 @@ class ListViewController: UIViewController {
     // A view that holds the place of the filtersView in the stack view
     private let filterPlaceholder = UIView()
 
+    private var allEateries: [Eatery] = []
```

Comment:
Do we need this? There is an `eateries` property defined below.

Reply:
we do need this because eateries actually refer to the ones in the list that are to be displayed. allEateries are all of the eateries. sorry for confusing logic fixed all of the heinous code it should be a lot better now